NSS-Day-Cohort-68/rare-client-rare-team-1
# Categories lmw

- Created CreateCategories module which links to 
- added link in Categories to /create-categories.
- This PR resolves https://github.com/NSS-Day-Cohort-68/rare-client-rare-team-1/issues/43

Directories added:
src/components/Categories
src/components/Welcome
src/components/Navbar
src/components/Views

Files added: 
src/components/Categories/Categories.js
src/components/Categories/CreateCategory.js
src/components/Categories/Categories.css
src/components/Welcome/Welcome.js
src/components/Welcome/Welcome.css
src/components/navbar/Navbar.js
src/components/navbar/Navbar.css
src/components/Views/ApplicationViews.js
src/components/Views/Authorized.js


Test With Expected User Input:

- After pulling the most recent version of this branch, run npm start from the rare-client-rare-team-1 directory.
go to http://localhost:3000/. You should see 3 buttons: Login, Register and Categories.
- Click on the categories link to go to http://localhost:3000/categories 
- You should see a button called "Create Category," click it.
- You should be directed to http://localhost:3000/create-category where you will see a text box next to text that says "Category" and a save button. 
- Without entering any text in the field, click SAVE and verify there is an error that no text was entered. 
- pull latest version of are-api-rare-team-1-1
- Open rare-api-rare-team-1-1 and run the debugger in json-server.py
- Back in http://localhost:3000/create-category add a test category label in the text box and click SAVE
- Navigate back to the rare-api-rare-team-1-1 file in vs code and click on the db.sqlite3 file. Refresh the database in the top left corner and verify a new label is created.
- go back to http://localhost:3000/ and click Login and verify there is the word login at the top of the page.
- go back to http://localhost:3000/ and click Register and verify you are taken to the previously made register page.
